testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/AbstractAdvancedBrokerTest.java

```diff
@@ -157,7 +156,7 @@ public void testAccountManagementLinkedIdentityAlreadyExists() {
         driver.navigate().to(getAccountUrl(getConsumerRoot(), bc.consumerRealmName()));
         logInWithBroker(bc);
         waitForAccountManagementTitle();
-        accountUpdateProfilePage.assertCurrent();
+        Assert.assertFalse(loginPage.isCurrent());
```

Comment:
Should we updated to login to test app, and check that is opened rather the checking login page isn't current page